README.md

```diff
@@ -1,4 +1,84 @@
-# caddy/caddy
+[![caddy/caddy on DockerHub][dockerhub-image]][dockerhub-url]
+[![Docker Build][gh-actions-image]][gh-actions-url]
 
-A [Caddy](https://caddyserver.com/) (version 2) Docker image.
+# [caddy/caddy](https://hub.docker.com/r/caddy/caddy)
 
+## Supported tags and respective `Dockerfile` links
+
+- [`v2.0.0-beta.15`, `v2.0.0-beta.15-alpine`, `alpine`, `latest`](https://github.com/caddyserver/caddy-docker/blob/f8db28380a31026b30d4fb361364ce88ec9abed1/alpine/Dockerfile)
+- [`v2.0.0-beta.15-slim`, `scratch`](https://github.com/caddyserver/caddy-docker/blob/f8db28380a31026b30d4fb361364ce88ec9abed1/scratch/Dockerfile)
```

Comment:
These will have to be updated at every release, it looks like?

Reply:
Yup - for official images this is handled by an update script. I'm going to look at auto-generating this later